sourcemash/frontend/static/js/routers/app_router.js

```diff
@@ -6,6 +6,7 @@ Sourcemash.Routers.AppRouter = Backbone.Router.extend({
         "categories/:id": "showCategory",
         "saved": "showSaved",
         "browse": "browseFeeds",
+        "categorizer": "showCategorizer",
```

Comment:
Here are the (eh) options as i see them:

1. Side nav: fine but its getting cluttered. 
2. Top nav: eh, don't love it 
3. Turn help menu button into a toolbox and put the help menu and categorizer in that: also don't love it
4. "Hide it" by making it link from only one page (eg splash) and the help menu: worst